core-developers/committing.rst

```diff
@@ -80,21 +80,14 @@ to enter the public source tree. Ask yourself the following questions:
 Updating NEWS and What's New in Python
 --------------------------------------
 
-Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
-If the change is particularly interesting for end users (e.g. new features,
-significant improvements, or backwards-incompatible changes), then an entry in
-the ``What's New in Python`` document (in ``Doc/whatsnew/``) should be added
-as well. Changes that affect documentation only generally do not require
-a ``NEWS`` entry.
+Changes that entail NEWS entries
```

Comment:
```suggestion
Changes that require NEWS entries
```
I would use simpler terms here.  I think "require" still fits even if the section also describes what changes *don't* require a NEWS entry.

Reply:
Sure, I'm fine with using 'require'.